src/duke/Ui.java

```diff
@@ -0,0 +1,301 @@
+package duke;
+import java.io.File;
+import java.lang.reflect.Array;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    boolean exited = false;
+    TaskList tl;
+    Storage s;
+    Ui(TaskList tl, Storage s) {
+        this.tl = tl;
+        this.s = s;
+    }
+
+    /**
+     * Starts the scanner and takes in the commands
+     */
+    public void start() {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____\n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo + "\n" + "I can be your friend who manages your task!");
+
+        try {
+            while (!exited) {
```

Comment:
Perhaps it would be better if your booleans can adhere to the coding guide:
e.g. isNotExited